cmd/shell_mgmt/include/shell_mgmt/shell_mgmt.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef H_SHELL_MGMT_
+#define H_SHELL_MGMT_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * Command IDs for shell management group.
+ */
+#define SHELL_MGMT_ID_EXEC   0
```

Comment:
Why not make it configurable like `SHELL_MGMT_MAX_LINE_LEN`?

Reply:
Thanks for the feedback! 

Perhaps I am misunderstanding something, but I don't believe this is up to the user to configure since it is a defined command id which is part of the shell management group.

I reviewed the protocol description [here](https://github.com/apache/mynewt-mcumgr/blob/master/protocol.md) and understood from it that the `shell exec` function in the mcumgr-cli would generate a frame header with `.nh_group = 9` and `.nh_id = 0`. I then setup the mapping for the actual implementation accordingly. If that isn't how it works I am not sure how I would map the function to the cli command without those being pre-defined and agreed upon by both the client and the manager.

I made the other options configurable because those do have repercussions if the os is setup differently than assumptions I would make. For instance, if I just assumed a max line length of 300, but the user configured zephyr with a max line length of 100 (or 400), things might not behave as expected.

Reply:
OK.